corehq/motech/repeaters/models.py

```diff
@@ -330,6 +331,23 @@ def to_json(self):
         repeater_dict.pop('is_deleted', None)
         repeater_dict.pop('next_attempt_at', None)
         repeater_dict.pop('last_attempt_at', None)
+
+        return self._make_serializable_dict(repeater_dict)
+
+    def _make_serializable_dict(self, repeater_dict):
```

Comment:
It could be surprising that this function mutates the dict that is passed as an argument, and then later returns that same (mutated) dict. This function violates the functional programming best practice that a function should not have side effects. However, it would also be wasteful to create a new dict in this case. What do you think of renaming it and changing the function to not return a value to make it clear that it is expected that the passed in dict will be mutated?
```suggestion
        self._convert_to_serializable(repeater_dict)
        return repeater_dict

    def _convert_to_serializable(self, repeater_dict):
```

Reply:
I was returning the `repeater_dict` and saving it in the variable to convey the idea that the value might be mutated. But I think your suggestion of renaming the function makes much more sense.